core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -63,6 +63,11 @@ trait RTS {
    */
   val threadPool: ExecutorService = newDefaultThreadPool()
 
+  /**
+   * The fiber's execution context.
+   */
+  lazy val executionContext = ExecutionContext.fromExecutor(threadPool)
```

Comment:
@ghostdogpr Can you turn this into `val` and not `lazy val` in a follow-up PR? `lazy val` will synchronize which will impose performance penalty.

Reply:
Oops, sure!